src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -305,12 +305,63 @@ public class MonoAOTCompiler : Microsoft.Build.Utilities.Task
     private FileCache? _cache;
     private int _numCompiled;
     private int _totalNumAssemblies;
+    private bool _collectTrimmingEligibleMethodsValueFromPreviousBuild;
 
     private readonly Dictionary<string, string> _symbolNameFixups = new();
     private static readonly char[] s_semicolon = new char[]{ ';' };
 
     private bool ProcessAndValidateArguments()
     {
+        string monoAotPropertyValuesFilePath = Path.Combine(IntermediateOutputPath, "monoAotPropertyValues.txt");
+
+        if (File.Exists(monoAotPropertyValuesFilePath))
```

Comment:
- move this to a separate method, which just returns the dictionary
- and here after the call, you can check for the specific property
- This would allow use of this mechanism for other properties in future

Reply:
done